src/store/reducers/auth.ts

```diff
@@ -51,3 +63,41 @@ export default function authReducer(state = initialState, { type, payload }) {
       return state;
   }
 }
+
+
+export  function confirmPasswordReducer(state:AuthState = initialState, { type, payload }) {
```

Comment:
Why do we have a separate reducer for `confirmPassword`?
Can you shed light on this please?

Reply:
Put the confirm password to authReducer